app/src/utils/test/setupTests.ts

```diff
@@ -1,5 +1,6 @@
 // Handle test setup here
 import '@testing-library/jest-dom';
+import 'whatwg-fetch';
```

Comment:
Yeah, this fixes the issue [here](https://github.com/reduxjs/redux-toolkit/issues/1271). Basically, the tests making requests to the server were failing in CI because there wasn't a reference to fetch in that environment, and this provides that.